Detectors/EMCAL/calibration/testWorkflow/emc-channel-calib-workflow.cxx

```diff
@@ -36,7 +36,8 @@ using namespace o2::emcal;
 void customize(std::vector<ConfigParamSpec>& workflowOptions)
 {
   std::vector<ConfigParamSpec> options{
-    {"configKeyValues", VariantType::String, "", {"Semicolon separated key=value strings"}}};
+    {"configKeyValues", VariantType::String, "", {"Semicolon separated key=value strings"}},
+    {"no-loadCalibParamsFromCCDB", VariantType::Bool, false, {"if true, load the EMCalCalibParams from the ccdb. If false, use the default values given in the struct"}}};
```

Comment:
I think the description is now swapped with respect to the program logics.

Reply:
Yes you are right. I corrected it. Thanks for catching that!